src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    double obtainArea();
+
+}

```

Comment:
It is more common to use `get`
remove the last empty line in all classes/interfaces/methods where it is present
```suggestion
public interface AreaCalculator {
    double getArea();
```